bs_detector/bias.py

```diff
@@ -475,27 +483,59 @@ def print_raw_data_for_features(list_of_sentences):
             print(feat.keys())
             KEYS_DONE = True
         print(feat.values())
-#print_raw_data_for_features(get_list_from_file('input_text_original'))
+
+
+# print_raw_data_for_features(get_list_from_file('input_text_original'))
 
 def compute_bias(sentence_text):
-    features = extract_bias_features(unicode(sentence_text, errors='ignore'))
+    if isinstance(sentence_text, unicode):
+        features = extract_bias_features(sentence_text)
+    else:
+        features = extract_bias_features(unicode(sentence_text, errors='ignore'))
```

Comment:
Should this unicode handling be built into the `extract_bias_features` function? It seems like it will be repeated every time you call `extract_bias_features`?

Reply:
Its related to a change with:
sentences = nltk.sent_tokenize(statement_text.decode("ascii", "ignore"))

where the text in the file is an unknown encoding and the sentences are being converted to ascii for sentence tokenization. So then afterwards, the sentence need to be converted back to unicode (with errors=ignore).

It looks like there is a unicode call within extract_bias_feature but not with 'errors=ignore'. If @cjhutto agrees I'll update that to use errors=ignore or let's discuss other thoughts or assumptions about inputs. One concern for the case of multi-sentence statements is to give the best input to the nltk sentence tokenization method.

Reply:
The unicode handling was moved to be built into the `extract_bias_features` function (for a similar reason mentioned -- it is needed for the nltk tokenization)...